cmd/chain-maind/app/app.go

```diff
@@ -84,7 +84,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 }
 
 func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
-	authclient.Codec = encodingConfig.Marshaler
+	// authclient.Codec = encodingConfig.Marshaler
```

Comment:
https://github.com/cosmos/cosmos-sdk/commit/d7dd1d7affc02a819845802da540bc2742d82074#diff-146f73b0327c426deb3126e44f9234c7a6fe190c1bcb4c98ab8366b3485922edR81
There also this change in new version, is that relevant?